heed/src/env.rs

```diff
@@ -126,7 +177,64 @@ impl EnvOpenOptions {
         self
     }
 
+    /// Specifies that the [`Env`] will be encrypted using the `A` algorithm with the given `key`.
+    ///
+    /// You can find more compatible algorithms on [the RustCrypto/AEADs page](https://github.com/RustCrypto/AEADs#crates).
+    ///
+    /// Note that you cannot use any type of encryption algorithm as LMDB exposes a nonce of 12 bytes.
```

Comment:
LMDB nonces are 16 bytes. The LMDB example truncates to 12 bytes.

Reply:
Whoops thank you, fixing the doc 👍